src/pages/candy-machine/sugar/commands/sign.md

```diff
@@ -16,4 +16,13 @@ And running with a specific keypair:
 
 ```
 sugar sign -k creator-keypair.json
+```
+
+Developers can provide a custom RPC URL with the command:
+```
+sugar sign -r <RPC_URL>
+```
+Note using `sugar sign` relies on a standard `getProgramAccounts` call to the Metaplex Token Metadata program (i.e., `metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s`). This is very inefficient. The recommended solution is to sign NFTs individually using the command:
```

Comment:
I think my preference would be towards more neutral language here. Otherwise this is great!

`Note using `sugar sign` relies on an inefficient `getProgramAccounts` call on the Metaplex Token Metadata program (i.e., `metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s`). The recommended solution is to sign NFTs individually using the command:`

Reply:
I've updated the PR to make its language more neutral with the following [commit](https://github.com/metaplex-foundation/developer-hub/pull/24/commits/d4054081a0626045bddd45f9d4c3fd6771e0811a). Let me know what you think!

Reply:
Nit: can you say "on the Token Metadata program" instead of "to the Token Metadata program" since gPA is not an instruction in the Token Metadata program?

Reply:
Sorry, just seeing this now - I've updated the wording @blockiosaurus